src/Data/String/CodePoints.purs

```diff
@@ -57,10 +58,27 @@ instance showCodePoint :: Show CodePoint where
 -- I would prefer that this smart constructor not need to exist and instead
 -- CodePoint just implements Enum, but the Enum module already depends on this
 -- one. To avoid the circular dependency, we just expose these two functions.
+-- |
+-- | ```purescript
+-- | >>> it = codePointFromInt 0x1D400 -- U+1D400 MATHEMATICAL BOLD CAPITAL A
+-- | Just (CodePoint 0x1D400) 
+-- | map singleton it
```

Comment:
On reflection, I think mixing the styles like this is potentially a bit confusing. What do you think? What about using the repl-style of examples with
```
>>> expr
output
```
throughout? I'm not super fussed about consistency throughout all the core libraries right now, but I think consistency within any given module is probably a good idea. Or, at the very least, consistency within particular examples.

Reply:
I like the 
```
>>> expr
output
```
style. It makes it easy to understand in which context the expressions are evaluated and makes it easy for the reader to test in their repl.

It also might allow for automated testing of the examples in the future.

Reply:
Yeah, that's pretty much what I was thinking too :)